BurnOutSharp/ProtectionType/SafeDisc.cs

```diff
@@ -25,6 +37,20 @@ public string CheckPortableExecutable(string file, PortableExecutable pex, bool
             if (!string.IsNullOrWhiteSpace(name) && name.Equals("SafeCast2", StringComparison.OrdinalIgnoreCase))
                 return $"SafeCast";
 
+            // Present on all "CLOKSPL.EXE" versions before SafeDisc 1.06.000. Found on Redump entries 61731 and 66004. 
+            // Only found so far on SafeDisc 1.00.025-1.01.044, but the report is currently left generic due to the generic nature of the check.
+            if (!string.IsNullOrWhiteSpace(name) && name.Equals("SafeDisc", StringComparison.OrdinalIgnoreCase))
+                return $"SafeDisc";
+
+            // TODO: This doesn't properly grab the File Description for secdrv.sys and I'm not sure why.
+            if (!string.IsNullOrWhiteSpace(name) && name.Equals("Macrovision SECURITY Driver", StringComparison.OrdinalIgnoreCase))
+                return $"SafeDisc Security Driver {GetSecDrvExecutableVersion(pex)}";
+
+            // Present on all CLOKSPL.DLL versions before SafeDisc 1.06.000. Found on Redump entries 61731 and 66004. 
```

Comment:
Why you quoted it before and not know, please be consistent.

Reply:
These will be addressed in a documentation update later.